README.md

```diff
@@ -17,11 +17,6 @@ using [TinyGo](https://github.com/tinygo-org/tinygo).
 TinyGo doesn't have full support of the Go Standard Library. However, this
 shouldn't pose a limit to policy authors.
 
-The biggest drawback of TinyGo is the limited (as of TinyGo v0.23) support
-of Go Reflection. Because of that the `encoding/json` package from the Standard
-Library is **not usable**. The code will compile just fine, but at **runtime**
-a panic will be occur.
```

Comment:
Not related about this specific change, but could you add a warning about having to use tinygo >= 0.28.1. Maybe this can be added right after  line 13